### PR TITLE
[doc](docs) Add some docs of baidu cloud bos

### PR DESCRIPTION
**Pull request by zxealous**
## Proposed changes

Issue Number: close #xxx

add bos doc

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zxealous** [comment]
run buildall

File: docs/en/docs/advanced/broker.md [hunk 1]
```diff
@@ -218,6 +219,17 @@ Same as Apache HDFS
 )
 ```
 
+#### Baidu Cloud OBS
+当前使用BOS时需要将[bos-hdfs-sdk-1.0.3-community.jar.zip](https://sdk.bce.baidu.com/console-sdk/bos-hdfs-sdk-1.0.3-community.jar.zip)下载并解压后把jar包放到broker的lib目录下。
```
Comment (luozenglin) [inline docs/en/docs/advanced/broker.md:223]: Please use English

**Comment by zxealous** [comment]
run buildall

**Review by luozenglin — APPROVED** [review]
LGTM

**Review by cambyzju — APPROVED** [review]
LGTM

**Pull request merged**